Give emergency air tanks 50/50 N2/O2 mix

Partmedia opened the pull request:
## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Emergency oxygen tanks spawn in survival boxes and can sometimes be found around the station. However, some species don't breathe oxygen.

Instead of adding content bloat, rename existing emergency _oxygen_ tanks to emergency _air_ tanks, preserving the prototype ID so that no changes are needed. Add a 50/50 N2/O2 mix to filled tanks and double regulator output so that users don't gasp by default. The amount of time that an emergency tanks lasts remains the same (double the gas, but also double the consumption rate).

While here, remove custom coloring from the large yellow oxygen tank. Large tanks should be consistently colored.

**Screenshots**
N/A

**Changelog**
:cl: notafet
- tweak: Emergency gas tanks now contain both oxygen and nitrogen.

Androidonator commented:
What if every race spawned with corresponding air tank and instead of emergency lockers we would have emergency dispensors that dispense things if they are authorized to do so

Partmedia commented:
While that's certainly a possibility, there was some discussion about how this would cause unnecessary content bloat.

0x6273 commented:
This would be inconsistent with the regular air tanks, since they use a different mix of mostly nitrogen.

Partmedia commented:
> This would be inconsistent with the regular air tanks, since they use a different mix of mostly nitrogen.

That's why they're colored differently. If we wanted to adjust the mixture to the standard air mix, oxygen-breathers would have significantly reduced emergency air capacity. Is this desirable?

A similar fix in #10981 used the same 50/50 mix.

0x6273 commented:
I think just adding emergency nitrogen tanks is the best solution.

Partmedia commented:
> I think just adding emergency nitrogen tanks is the best solution.

Can you give some arguments in favor of this?

plinyvic commented:
halving the effective capacity of tanks sounds like a harsh solution. would probably  make more sense to just have nitrogen internals tanks.
ss13 plasmamen already had their own internals tank anyway.

Partmedia commented:
> halving the effective capacity of tanks sounds like a harsh solution. would probably make more sense to just have nitrogen internals tanks. ss13 plasmamen already had their own internals tank anyway.

As I stated in the PR description, the capacity is not halved.

retequizzle commented:
why does it matter, 90% of you will throw your emergency air away whether it's all O2, all N2, a 50/50 mix or otherwise. amount of bikeshedding being done for this when most of you abandon it anyway is dumb

Kromabae commented:
Just make tank dispensers public access, that way everyone can have a tank that lasts 2 hours

civilCornball commented:
It’s a good idea, but might confuse some players.
“Why am I gasping? My tank still has air!”

Partmedia commented:
> It’s a good idea, but might confuse some players. “Why am I gasping? My tank still has air!”

As I stated in the PR description, players will not gasp because I've already adjusted the default regulator pressure. Can you folks please read the PR description in case I've already addressed your concerns?

mirrorcult commented:
im good with this in cocnept but the emergency tank needs to be made not blue (preferably white like air canisters) since blue implies oxygen

Partmedia commented:
Overcome by events.
